fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -346,6 +346,28 @@ ACTOR Future<Void> getRate(UID myID,
 	}
 }
 
+void proxyGRVThresholdExceeded(const GetReadVersionRequest* req, GrvProxyStats* stats) {
+	++stats->txnRequestErrors;
+	// FIXME: send an error instead of giving an unreadable version when the client can support the error:
+	// req.reply.sendError(proxy_memory_limit_exceeded());
```

Comment:
Do you have any interest in addressing these in this series of PRs, or should we file an issue?

Reply:
Done.